dstep/translator/IncludeHandler.d

```diff
@@ -109,12 +121,39 @@ class IncludeHandler
         imports ~= "core.stdc.config";
     }
 
+    @property{
+        /// Makes includes that match regex filter be converted to import with prefix.
+        void autoImportPrefix (string prefix)
+        {
+            this.convertIncludes = true;
+            this.importPrefix = prefix;
+        }
+
+        string autoImportPrefix ()
+        {
+            return this.importPrefix;
+        }
+
+        /// Makes includes that match regex filter be converted to import with prefix.
+        void autoImportFilter (string filter)
+        {
+            this.convertIncludes = true;
+            this.convertableIncludePattern = regex(filter);
+        }
+
+        string autoImportFilter()
+        {
+            return this.convertableIncludePattern.stringof;
```

Comment:
I'm pretty sure this will return `"convertableIncludePattern"` and not the actual regular expression. I haven't been able to find how to extract a string representation of a regular expression, but you could store the string separately.